services/ui-src/src/routes/Profile/Profile.tsx

```diff
@@ -51,11 +65,17 @@ const sx = {
       marginTop: "3.5rem",
     },
   },
-  heading: {
-    marginBottom: "2rem",
+  headerText: {
+    marginBottom: "rem",
```

Comment:
Still need the bottom rem on this page to push the footer down.

```suggestion
    marginBottom: "2rem",
```